test/PowerShellEditorServices.Test/PowerShellEditorServices.Test.csproj

```diff
@@ -34,8 +34,7 @@
     <PackageReference Include="xunit" Version="2.4.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" />
     <PackageReference Include="Xunit.SkippableFact" Version="1.4.13" />
-    <PackageReference Include="OmniSharp.Extensions.LanguageServer" Version="0.19.2" />
-    <DotNetCliToolReference Include="dotnet-xunit" Version="2.4.0-beta.1.build3958" />
```

Comment:
Not sure what on earth this `DotNetCliToolReference` was for, it doesn't seem needed any more.